test/DynamoCoreTests/NodeToCodeTest.cs

```diff
@@ -976,8 +976,7 @@ public void NonUniqueNamespaceConflict_on_node2code_doesNotCrash()
             var cbn = nodes.OfType<CodeBlockNodeModel>().FirstOrDefault();
             Assert.IsNotNull(cbn);
 
-            var error = "Multiple definitions for 'FFITarget.B.DupTargetTest' are found as FFITarget.C.B.DupTargetTest, FFITarget.B.DupTargetTest";
```

Comment:
@aparajit-pratap is there still a test that asserts this error is raised in the correct situation where there actually are conflicts?

Reply:
Modified an existing test to catch this scenario.